adafruit_midi/__init__.py

```diff
@@ -54,13 +58,13 @@ class MIDI:
 
     def __init__(
         self,
-        midi_in=None,
-        midi_out=None,
+        midi_in: Optional[Any] = None,
+        midi_out: Optional[Any] = None,
```

Comment:
Instead of `Any` I think `midi_in` can be `usb_midi.PortIn` and `midi_out` can be `usb_midi.PortOut` types respectively. https://docs.circuitpython.org/en/latest/shared-bindings/usb_midi/index.html#usb_midi.PortIn

Reply:
the correct type might be typing.BinaryIO

Reply:
changed to `typing.BinaryIO` in the latest commit.